### PR TITLE
DOC: Added "Raises `KeyError`"

### PR DESCRIPTION
**Pull request by ShaharNaveh**
- [ ] closes #xxxx
- [ ] tests added / passed
- [x] passes `black pandas`
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: pandas/compat/chainmap.py [hunk 1]
```diff
@@ -6,12 +6,18 @@ def __setitem__(self, key, value):
         for mapping in self.maps:
             if key in mapping:
                 mapping[key] = value
-                return
+                break
```
Comment (jbrockmendel) [inline pandas/compat/chainmap.py]: does break vs return make a difference?  here or below?

Comment (MomIsBestFriend) [reply pandas/compat/chainmap.py]: > does break vs return make a difference? here or below?

No it doesn't (not here at least), But I think it's making the code more readable.
I don't mind reverting those changes if you think they're doing the opposite.

Comment (WillAyd) [reply pandas/compat/chainmap.py]: Doesn't this always now run line 10 whereas before it wouldn't previously if the function returned here?

**Review by WillAyd — CHANGES_REQUESTED** [review]
Not sure about these changes - is this purely a stylistic thing you are aiming for?

**Comment by MomIsBestFriend** [comment]
> Not sure about these changes - is this purely a stylistic thing you are aiming for?

Yes, indeed purely stylistic change, I'll revert.

File: pandas/compat/chainmap.py (hunk 1 above)
Comment (MomIsBestFriend) [reply pandas/compat/chainmap.py]: \*pacepalm\* Yes you're 100% right.

**Comment by jreback** [comment]
thanks @MomIsBestFriend

**Pull request merged**